main/inc/lib/message.lib.php

```diff
@@ -1466,19 +1487,35 @@ public static function showMessageBox($messageId, $type)
 
         switch ($type) {
             case self::MESSAGE_TYPE_OUTBOX:
-                $message_content .= '<a href="outbox.php?'.$social_link.'">'.
-                    Display::return_icon('back.png', get_lang('ReturnToOutbox')).'</a> &nbsp';
-                $message_content .= '<a href="outbox.php?action=deleteone&id='.$messageId.'&'.$social_link.'" >'.
-                    Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
-                break;
+                    $message_content .= '<a href="outbox.php?' . $social_link . '">' .
+                        Display::return_icon('back.png', get_lang('ReturnToOutbox')) . '</a> &nbsp';
+                    $message_content .= '<a href="outbox.php?action=deleteone&id=' . $messageId . '&' . $social_link . '" >' .
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 126 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)